### ISSUE TITLE
How to load best model with same accuracy?

### ISSUE THREAD

**Issue by Min-seong-son** [keras-team/autokeras#926]
Hi :)

I want to load best model with same accuracy. (ImageClassifier)

So, In Training Part, 

**clf = ak.ImageClassifier( directory='/home/son/PycharmProjects/Project/SkinModel20192/')
clf.fit(train_input, train_label, validation_split=(1/9))**

And In Load Part,

**autokeras_model = tf.keras.models.load_model('/home/son/PycharmProjects/Project/SkinModel2019/my_skinmodel2019.h5')**

But After Running Load Part, I got lower Accuracy result than Training result... 

And I found one way of increasing accuracy, 

**autokeras_model.fit(train_input, train_label, validation_split=1/9, batch_size=32)**     

That is retraining in Load part using Same Train data set. As a result, It had a good effect. But I think It is inefficient way. I want to get same accuracy without retraining.

So, I want to know the **reason of problems (e.g : can not load hyperparameter or weights) and solution.**
Also, After exporting best model, Some files (oracle.json, trail.json, checkpoint etc / The picture is uploaded) are generated.. But I do not know how to use this file when I load the best model..

I would be happy if somebody could provide me with some help.

![스크린샷, 2020-01-25 18-39-05](https://user-images.githubusercontent.com/60286567/73119219-00451280-3fa2-11ea-9f8a-86f8d83c2806.png)
![스크린샷, 2020-01-25 18-38-34](https://user-images.githubusercontent.com/60286567/73119220-00dda900-3fa2-11ea-984a-399a5940da7b.png)

**Comment by haifeng-jin** [comment]
@Min-seong-song I believe this is because the processor is not exported. Tensorflow 2.1 has just released the preprocessing layers which enables a Keras model to contain preprocessors. We will support it in the next release of AutoKeras. Thank you

**Comment by Min-seong-son** [comment]
Thank you and I hope next version will be released ASAP :)
